### ISSUE TITLE
Zone 2 can't power off

### ISSUE THREAD

**Issue by musician2000** [miracle2k/onkyo-eiscp#36]
Hi 

First of all, love the script, use it a lot! 

I couldn't work out how to turn off zone 2?

I can do 

onkyo zone2.power=on

Which works great, but 

onkyo zone2.power=off
Error: "off" is not a valid argument for command "power" in zone "zone2"

at the moment I just using the muting command to work round this issue, but eventually would prefer to turn off the zone 2 in my script.

**Comment by xydix** [comment]
Try onkyo zone2.power=standby

**Comment by toshibochan** [comment]
did work? if work please close this issue

**Comment by Grandma-Betty** [comment]
Yes, `onkyo zone2.power=standby` works perfectly. Just tested right now on my TX-NR6100.
Someone should close this issue.

**Comment by FauthD** [comment]
+1 for the TX-NR509.
